tests/local_contract_calls.py

```diff
@@ -91,8 +91,10 @@ class ReceiptAggregateVoucher(EIP712Message):
 except ContractCustomError as e:
     print("Custom Error: %s" % e)
     print(decode_custom_error(erc20_abi_json, str(e), w3))
+    sys.exit(1)
```

Comment:
Just let the exception propagate all the way up. (applies everywhere you used this)
```suggestion
    raise
```